content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: 'Upgrade to Prisma 5'
+metaTitle: 'Upgrade to Prisma 5'
+metaDescription: 'Guides on how to upgrade to Prisma 5'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma 5.0.0 introduces a number of **breaking changes**. This guide explains how upgrading might affect your application and gives instructions on how to handle any breaking changes.
+
+</TopBlock>
```

Comment:
It would be great to still give an outlook on the _positive_ changes of Prisma 5, and potentially plan to link to the blog post we are going to publish. 

There are also other changes beyond the breaking ones, which are only listed in the release notes (bottom list of links), so maybe linking over there would also be good?